packages/ui/src/ui-component/dropdown/AsyncDropdown.jsx

```diff
@@ -152,6 +152,7 @@ export const AsyncDropdown = ({
                             )
                         }}
                         sx={{ height: '100%', '& .MuiInputBase-root': { height: '100%' } }}
+                        key={`${nodeData.id}-${name}-${nodeData.inputs[internalValue]}`}
```

Comment:
there are other places that uses AsyncDropdown, for example when creating custom assistant. In those cases, there will be no nodeData, then it will throw error:

![image](https://github.com/user-attachments/assets/9d670d3d-fbe0-40ee-b4da-2cbd68596c17)